Example/PaymentSheet Example/PaymentSheet Example/ExampleWalletButtonsView.swift

```diff
@@ -62,6 +91,7 @@ class ExampleWalletButtonsModel: ObservableObject {
                 configuration.customer = .init(
                     id: customerId, ephemeralKeySecret: customerEphemeralKeySecret)
                 configuration.returnURL = "payments-example://stripe-redirect"
+                configuration.willUseWalletButtonsView = true
```

Comment:
Just a thought -- since `willUseWalletButtonsView` only applies to FlowController -- would it be better to pass this (and potentially other configs) through `PaymentSheet.FlowController.create()` instead of `PaymentSheet.Configuration" ?

Reply:
I think it's a configuration option (even if it does only apply to FlowController), it seems odd to clutter up create with FC-specific options. It should support non-FC someday, and there are other options that don't apply to all PaymentSheets (e.g. `paymentMethodLayout` doesn't work for Embedded).

Reply:
I was considering something like the following in case we wanted to separate out this specific use case for flow controller. Not sure if this it's warranted though.
```
FlowController.create(
    intentConfig:  PaymentSheet.IntentConfiguraiton,
    configuration: PaymentSheet.Configuration,
    walletConfiguration: PaymentSheet.WalletConfiguration,
    completion:
)
```